_licenses/cddl-1.0.txt

```diff
@@ -0,0 +1,378 @@
+---
+title: Common Development and Distribution License 1.0
+spdx-id: CDDL-1.0
+
+description:
```

Comment:
Modifying the [wikipedia description](https://en.wikipedia.org/wiki/Common_Development_and_Distribution_License)

```suggestion
description: A weak copyleft license used by Sun Microsystems based on the Mozilla Public License. Files licensed under the CDDL can be combined with files licensed under other licenses, both open source and proprietary, so long as any code under the CDDL is made available.
```